extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -1121,9 +1155,6 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
     try {
       await loginToAllAccounts(password);
     } catch (e) {
-      captureException(
-        `Could not login when confirming password: ${JSON.stringify(e)}`,
-      );
```

Comment:
same change as above - don't report to Sentry every time a user enter's an incorrect password